sashimi/hardware/cameras/__init__.py

```diff
@@ -1,12 +1,14 @@
 from sashimi.config import read_config
 import time
 import numpy as np
+from sashimi.hardware.cameras.interface import AbstractCameraInterface
 
 conf = read_config()
 
 
-class BasicCamera:
+class BasicCamera(AbstractCameraInterface):
     def __init__(self):
+        super().__init__()
         self.encoding = "utf-8"
```

Comment:
What is this?

Reply:
I don´t really know. It was part of the old wrapper and not even sure what it does

Reply:
It definitely doesn't do anything for any camera, just remove it.